client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -96,15 +102,22 @@ const CardActionsRow = styled.div`
   }
 `
 
+const numericValidationRule = {
+  message: 'Can only contain numeric characters',
+  value: /^[0-9]+$/,
+}
```

Comment:
`type="number"` still allows users to enter `.` and `e` for decimals and scientific notation, respectively. This validation rule guarantees integer input